pkg/sql/rowexec/columnbackfiller.go

```diff
@@ -124,16 +128,16 @@ func (cb *columnBackfiller) runChunk(
 			// waiting for consistency when backfilling a column on GLOBAL tables.
 			commitWaitFn = txn.DeferCommitWait(ctx)
 
-			// TODO(knz): do KV tracing in DistSQL processors.
 			var err error
 			key, err = cb.RunColumnBackfillChunk(
 				ctx,
 				txn,
 				cb.desc,
 				sp,
 				chunkSize,
-				true,  /*alsoCommit*/
-				false, /*traceKV*/
+				updateRunThresholdBytes,
+				true, /*alsoCommit*/
+				cb.flowCtx.TraceKV,
```

Comment:
I figure I'd leave this change in, as the debugger assures me that this flag is set when `SET tracing = on,kv`.